### DISCUSSION TITLE
Gannet Connection Issue

### DISCUSSION THREAD

**Discussion by jdduprey** [sr320/course-fish546-2021#57] [Q&A]
I am getting the following error when I try to connect to Gannet, possibly because I typed the password in wrong initially?
`ssh casowari@gannet.fish.washington.edu`
`kex_exchange_identification: read: Connection reset by peer`

**Comment by kubu4** [comment]
> possibly because I typed the password in wrong initially?

Most likely this is the case. You only get three chances (within a 5 minute time frame) to get the password correct. After that, the server blocks your IP address.

Could you please post your IP address and I'll unblock it? To get  your IP address, type "What's my IP" using the Google search engine in any web broswer. If you don't want to post the IP address here, you can send me an email: samwhite<at>uw<dot>edu (replace the `<at>` and the `<dot>` with the appropriate symobls.

**Reply by jdduprey** [reply]
Sent, damn these fat fingers...
